apps/mobile/src/app/(home)/settings/index.tsx

```diff
@@ -100,6 +100,58 @@ export default function SettingsScreen() {
             onPress={() => router.navigate(AppRoutes.SettingsNetworks)}
             testID={TestId.settingsNetworkButton}
           />
+          <SettingsListItem
```

Comment:
This is a mistake I think?

Reply:
Indeed. Thanks for catching. Probably meant to test the scrolling and committed by mistake.